app/src/features/permits/CreatePermitPage.tsx

```diff
@@ -65,20 +65,24 @@ export interface IPermitsArrayItem {
 }
 
 export interface ICreatePermitsForm {
-  first_name: string;
-  last_name: string;
-  email_address: string;
-  coordinator_agency: string;
-  share_contact_details: string;
+  coordinator: {
+    first_name: string;
+    last_name: string;
+    email_address: string;
+    coordinator_agency: string;
+    share_contact_details: string;
+  };
   permits: IPermitsArrayItem[];
 }
 
 export const PermitsInitialValues: ICreatePermitsForm = {
-  first_name: '',
-  last_name: '',
-  email_address: '',
-  coordinator_agency: '',
-  share_contact_details: 'false',
+  coordinator: {
+    first_name: '',
+    last_name: '',
+    email_address: '',
+    coordinator_agency: '',
+    share_contact_details: 'false'
+  },
```

Comment:
I don't think these changes are used anywhere? Can we revert them, only to reduce conflicts with the other permit branch if possible

Reply:
@KjartanE I ended up reverting this change. I was thinking this was the project permit page, not the stand alone permit page.  Tests broke when these got removed, so easier to just put it back for now, and Alfred can rip it out in his ticket which is removing this functionality entirely anyways.